h/services/list_groups.py

```diff
@@ -81,34 +119,31 @@ def _open_groups(self, user=None, authority=None):
                       .all())
         return self._sort(groups)
 
-    def _private_groups(self, user=None):
-        """Return this user's private groups per user.groups."""
+    def _user_groups(self, user=None):
+        """Return all groups that this user is a member of regardless of type"""
 
         if user is None:
             return []
         return self._sort(user.groups)
```

Comment:
An important concept to grasp is that `user.groups` can now include `restricted` groups along with `private` groups — **if** the user is a member of the given `restricted` group.

Reply:
What about if the user is the creator of an open group?

Reply:
Short answer per this PR: nope—the open group's creator is not a "member" relationship and so those groups are not in a user's `groups`. We should open a new issue to track this requirement.